PowerPointLabs/PowerPointLabs/SyncLab/SyncFormatConstants.cs

```diff
@@ -31,9 +32,37 @@ public static FormatTreeNode[] FormatCategories
             }
         }
 
+        public static List<Format> Formats
+        {
+            get
+            {
+                List<Format> list = new List<Format>();
+                list.AddRange(GetFormatsFromFormatTreeNode(FormatCategories));
+                return list;
+            }
+        }
+        
+        private static Format[] GetFormatsFromFormatTreeNode(FormatTreeNode[] nodes)
```

Comment:
Don't think I should go into detail since the FormatTreeNode object is known to build a Tree. (the reader can refer to the FormatTreeNode class for more info)